babel.config.js

```diff
@@ -44,7 +44,7 @@ module.exports = function (api) {
       [
         require('@babel/plugin-proposal-class-properties').default,
         {
-          loose: true
+          loose: false
```

Comment:
Wasn't sure what this was so looked it up, made my head spin a bit but it seems like turning loose mode off is overall the better option in the long run? Also we don't have loads of JS so don't need to worry about minor slowdowns. So this looks good 👍

Reply:
> turning loose mode off is overall the better option

🆒 that was my finding too!